sklearn/cluster/optics_.py

```diff
@@ -425,6 +422,42 @@ def fit(self, X, y=None):
 
     # OPTICS helper functions
 
+    def _calculate_core_distances_(self, X, nbrs, working_memory=None):
+        """Compute the k-th nearest neighbor of each sample
+
+        Equivalent to nbrs.kneighbors(X, self.min_samples)[0][:, -1]
+        but with more memory efficiency.
+
+        Parameters
+        ----------
+        X : array, shape (n_samples, n_features)
+            The data.
+        nbrs : NearestNeighbors instance
```

Comment:
I would rather use "neighbors" instead of "nbrs" that parses as "numbers" in my brains.